### DISCUSSION TITLE
Unminified version of opensourcepos.min.js

### DISCUSSION THREAD

**Discussion by hagopmanougian** [opensourcepos/opensourcepos#4140] [Q&A]
Hi everybody
Where or how can I get the original js code of the file opensourcepos.min.js ?
Thanks in advance

**Comment by jekkos** [comment]
You can get that live by adding `?debug=true` to your page, it should then load all original code

**Comment by hagopmanougian** [comment]
Thanks for the reply. You've done a great job. The software is very stable in contrast to other packages that are full of bugs.
One simple question. Where do I add the ?debug=true ?
